src/modules/client/components/customers/CustomerProfileHeader.vue

```diff
@@ -97,11 +134,49 @@ export default {
         'dot dot-archived': value === ARCHIVED,
       };
     },
+    async refreshCustomer () {
+      try {
+        await this.$store.dispatch('customer/fetchCustomer', { customerId: this.customer._id });
+      } catch (e) {
+        console.error(e);
+        NotifyNegative('Erreur lors du chargement des données.');
+      }
+    },
+    setStoppingDateErrorMessage (validations) {
+      if (!validations.stoppedAt.minDate) return 'La date d\'arrêt ne peut être antérieure à la date de création.';
+
+      return REQUIRED_LABEL;
+    },
+    resetStopSupportModal () {
+      this.$v.newStatus.$reset();
+      this.newStatus = { stopReason: '', stoppedAt: '' };
+    },
+    async stopSupport () {
+      try {
+        this.modalLoading = true;
+        const payload = {
+          stoppedAt: getEndOfDay(new Date(this.newStatus.stoppedAt)),
```

Comment:
idem tu  peux simplifier `getEndOfDay(this.newStatus.stoppedAt)`

Reply:
same